__tests__/nc_news.test.js

```diff
@@ -86,3 +86,42 @@ describe('GET /api/articles/:article_id', () => {
 			});
     });
 });
+
+
+describe('POST, /api/articles/:article_id/comments', () => {
+    it('POST:201 should return correct status code', () => {
+        return request(app).post('/api/articles/1/comments')
+        .send({
+            username: "icellusedkars",
+            body: "Enjoyed it"
+        })
+        .expect(201);
+    });
+    it('POST:201 should post new comment and return comment with confirmation', () => {
+        return request(app).post('/api/articles/1/comments')
+        .send({
+            username: "icellusedkars",
+            body: "Enjoyed it"
+        })
+        .then(({body}) => {
+            expect(body.comment).toHaveProperty("comment_id");
+            expect(body.comment).toHaveProperty("body");
+            expect(body.comment).toHaveProperty("article_id");
+            expect(body.comment).toHaveProperty("author");
+            expect(body.comment).toHaveProperty("votes");
+            expect(body.comment).toHaveProperty("created_at");
+
```

Comment:
I'd consider adding an additional test that ignores unnecessary properties for example:
```
{
            username: "icellusedkars",
            body: "Enjoyed it",
            extra: "extra property"
        }
```
What status would this respond with?